mteb/tasks/Classification/vie/AmazonCounterfactualVNClassification.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+
+
+class AmazonCounterfactualVNClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="AmazonCounterfactualVNClassification",
+        dataset={
+            "path": "GreenNode/amazon-counterfactual-vn",
+            "revision": "b48bc27d383cfca5b6a47135a52390fa5f66b253"
+        },
+        description=(
+            "A collection of Amazon customer reviews annotated for counterfactual detection pair classification."
```

Comment:
Please also add a description of how it was machine translated, and that it was adapted from AmazonCounterfactualClassification.